### PR TITLE
Created Button Component

### PR DESCRIPTION
**Pull request by khyatibaria**
### Button Component
Please Review it
image

### CODE REVIEW

**Comment by JatuFaique** [comment]
The styles look very good and clean.

File: button/button.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  --success-bgcolor: #a8f5d9;
+  --danger-bgcolor: #f9c0c0;
+  --warning-bgcolor: #fcffa6;
+  --padding-md: 1rem;
+  --margin-md: 1rem;
+  --fontSize: 1rem;
+  --radius: 0.25rem;
+}
+
+.conatiner {
```
Comment (zabihshaik) [inline button/button.css]: May be a typo in spelling 'container'

File: button/button.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  display: flex;
+  direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: baseline;
+  width: 700px;
```
Comment (zabihshaik) [inline button/button.css]: Better to use relative units like rems instead of fixed units like px

File: button/button.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Image Component</title>
+    <link rel="stylesheet" type="text/css" href="button.css" />
+    <script src="https://kit.fontawesome.com/99b4ab4b7a.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="conatiner">
```
Comment (zabihshaik) [inline button/button.html]: May be a typo in spelling 'container' as it should be reusable to others.

File: button/button.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
+<html>
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Image Component</title>
```
Comment (zabihshaik) [inline button/button.html]: Better to add meta tag with your name in content

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a few suggestions:
1. Please check for typos
2. Better to use rems instead of px
3. Better to add meta tag with your name in content

**Review by JatuFaique — COMMENTED** [review]
The Button styles looks clean to me. Good to go.

**Pull request merged**